package.json

```diff
@@ -83,7 +83,7 @@
     "strip-css-comments": "^3.0.0",
     "ts-jest": "^26.5.4",
     "ts-node": "^9.1.1",
-    "typescript": "~4.1.5",
+    "typescript": "^3.9.6",
```

Comment:
I'd prefer to make things work with the newer typescript rather than pin to 3.x if possible

Reply:
I agree on principle @Page- , but the TS bump has caused a weird `window.matchMedia is undefined` error in storybook, so it's not a TS compilation issue thing.
I do see some profit on getting master to have its tests passing asap and fix the storybook issue in a follow-up PR.